javascript/javascript1/week2/event_application.js

```diff
@@ -0,0 +1,16 @@
+function getEventWeekday(days) {
+    if (days < 0){
+        alert('Days can not be negative! Be positive :)');
+        return '';
+    }
+    let weekDays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+    let d = new Date();
```

Comment:
always use const until it doesn't let you

Reply:
fixed